src/code/sys_matrix.c

```diff
@@ -973,165 +998,166 @@ void func_800D2A34(MtxF* mf, f32 arg1, f32 arg2, f32 arg3, f32 arg4) {
     mf->xz = 0.0f;
     mf->yz = 0.0f;
     mf->wz = 0.0f;
-    mf->xx = arg1;
-    mf->yy = arg1;
-    mf->zz = arg1;
-    mf->xw = arg2;
-    mf->yw = arg3;
-    mf->zw = arg4;
+    mf->xx = scale;
+    mf->yy = scale;
+    mf->zz = scale;
+    mf->xw = translateX;
+    mf->yw = translateY;
+    mf->zw = translateZ;
     mf->ww = 1.0f;
 }
 
-void func_800D2A98(Mtx* mtx, f32 arg1, f32 arg2, f32 arg3, f32 arg4) {
+void Matrix_SetTranslateUniformScaleMtx(Mtx* mtx, f32 scale, f32 translateX, f32 translateY, f32 translateZ) {
```

Comment:
You use the convention:
`Matrix_SetTranslateUniformScaleMtx`
`Matrix_SetTranslateUniformScaleMtx2`
for this set of equivalent functions, but then use:
`Matrix_SetTranslateScaleMtx1`
`Matrix_SetTranslateScaleMtx2`
For the set of equivalent functions below. I wonder if they should be consistent. Either `Mtx/Mtx2` or `Mtx1/Mtx2`. I don't have a preference either way

Reply:
I think it's because `Matrix_SetTranslateUniformScaleMtx` is the Mtx version of the MtxF function `Matrix_SetTranslateUniformScaleMtxF` (just a wrapper)